onnxmltools/convert/common/model_util.py

```diff
@@ -90,6 +90,14 @@ def make_model(name, ir_version, producer, producer_version, domain, model_versi
     model.domain = domain
     model.model_version = model_version
     model.doc_string = doc_string
+    # Neural network operators' version
+    opset = model.opset_import.add()
+    opset.domain = ''
+    opset.version = 3
+    # Traditional ML operators' version
```

Comment:
This should be conditional and added only to models that actually use ops from that namespace.

Reply:
Thanks. Now we collect the operator set information from existing operators only.